src/index.html

```diff
@@ -12,7 +13,109 @@
       href="./style.css"
     />
   </head>
+  <!--#endregion-->
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
```

Comment:
This block of code is repeated multiple times with only minor changes. Consider using a templating system or a loop in a scripting language to generate these elements programmatically. This will make your code more maintainable and less prone to errors.